applications/luci-app-filemanager/htdocs/luci-static/resources/view/system/filemanager.js

```diff
@@ -0,0 +1,1980 @@
+            'class': 'cbi-tab',
+            'style': 'display:block;'
+        }, [E('div', {
+            'id': 'file-list-container',
+            'class': 'resizable',
+            'style': 'width: ' + config.windowSizes.width + 'px; height: ' + config.windowSizes.height + 'px;'
+        }, [E('table', {
+            'class': 'table',
+            'id': 'file-table'
+        }, [E('thead', {}, [E('tr', {}, [E('th', {
+            'data-field': 'name'
+        }, [_('Name'), E('button', {
+            'class': 'sort-button',
+            'data-field': 'name',
+            'title': _('Sort by Name')
+        }, '↕'), E('div', {
+            'class': 'resizer'
+        })]), E('th', {
+            'data-field': 'type'
+        }, [_('Type'), E('button', {
+            'class': 'sort-button',
+            'data-field': 'type',
+            'title': _('Sort by Type')
+        }, '↕'), E('div', {
+            'class': 'resizer'
+        })]), E('th', {
+            'data-field': 'size'
+        }, [_('Size'), E('button', {
+            'class': 'sort-button',
+            'data-field': 'size',
+            'title': _('Sort by Size')
+        }, '↕'), E('div', {
+            'class': 'resizer'
+        })]), E('th', {
+            'data-field': 'mtime'
+        }, [_('Last Modified'), E('button', {
+            'class': 'sort-button',
+            'data-field': 'mtime',
+            'title': _('Sort by Last Modified')
+        }, '↕'), E('div', {
+            'class': 'resizer'
+        })]), E('th', {}, [E('input', {
+            'type': 'checkbox',
+            'id': 'select-all-checkbox',
+            'style': 'margin-right: 5px;',
+            'change': this.handleSelectAllChange.bind(this)
+        }), _('Actions')])])]), E('tbody', {
+            'id': 'file-list'
+        })])]), E('div', {
+            'id': 'status-bar'
+        }, [E('div', {
+            'id': 'status-info'
+        }, _('No file selected.')), E('div', {
+            'id': 'status-progress'
+        })]), E('div', {
+            'class': 'cbi-page-actions'
+        }, [E('button', {
+            'class': 'btn action-button',
+            'click': this.handleUploadClick.bind(this)
+        }, _('Upload File')), E('button', {
+            'class': 'btn action-button',
+            'click': this.handleMakeDirectoryClick.bind(this)
+        }, _('Make Directory')), E('button', {
+            'class': 'btn action-button',
+            'click': this.handleCreateFileClick.bind(this)
+        }, _('Create File')), E('button', {
+            'id': 'delete-selected-button',
+            'class': 'btn action-button',
+            'style': 'display: none;',
+            'click': this.handleDeleteSelected.bind(this)
+        }, _('Delete Selected'))])]), E('div', {
+            'id': 'content-editor',
+            'class': 'cbi-tab',
+            'style': 'display:none;'
+        }, [E('p', {}, _('Select a file from the list to edit it here.')), E('div', {
```

Comment:
Can this be displayed on the condition that we are not editing a file?

Reply:
Yes, it can and it does

Reply:
![Screenshot 2024-11-19 at 17 15 20](https://github.com/user-attachments/assets/fe3e2278-08ad-47df-993f-0ca00c202060)

Reply:
Done

Reply:
Done

Reply:
Fixed

Reply:
I can't resolve this conversation